tests/expected/example-complex-forecast-hub/forecasts/wk-inc-flu-hosp_01_2022-10-22.json

```diff
@@ -22,5 +22,13 @@
     "q0.5": [118, 175],
     "q0.75": [133, 193],
     "q0.975": [165, 233]
+  },
+  "Test-NumericOnly": {
+    "target_end_date": ["2022-10-29", "2022-11-05"],
+    "q0.025": [1542, 1443],
+    "q0.25": [1542, 1443],
+    "q0.5": [1724, 1724],
+    "q0.75": [1906, 2006],
+    "q0.975": [2334, 2562]
   }
 }
```

Comment:
for this file and the next one, have the "Test-NumericOnly" additions been hand-checked for correctness?